Application/src/main/java/control/DettagliEventoServlet.java

```diff
@@ -0,0 +1,50 @@
+package control;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.util.Collection;
+import java.sql.PreparedStatement;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import model.ConDB;
+import model.evento.Evento;
+import model.evento.EventoService;
+import model.utente.*;
+
+@WebServlet("/DettagliEvento")
+public class DettagliEventoServlet extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    
+    private EventoService eventoService = new EventoService(); 
+    private PrenotazioneService prenotazioneService = new PrenotazioneService();
+
+    public DettagliEventoServlet() {
+        super();
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String idEvento = request.getParameter("id");	//ID Evento passato nell'url
```

Comment:
nn lo chiamerei id ma id_evento.. crea molta confusione, però in effetti la servlet è dettagli evento.. vedi tu . penso che nella jsp nn scriverà id

Reply:
Anch'io avevo questo dubbio, però col fatto che la servlet si chiama DettagliEvento non dovrebbe essere ambigua come cosa, perchè poi visiti "/DettagliEvento?id=123", quindi si capisce che è riferito all'evento credo